examples/platforms/simulation/radio.c

```diff
@@ -1083,6 +1119,10 @@ void radioProcessFrame(otInstance *aInstance)
     otEXPECT_ACTION(otMacFrameDoesAddrMatch(&sReceiveFrame, sPanid, sShortAddress, &sExtAddress),
                     error = OT_ERROR_ABORT);
 
+    iid = getIidFromFrame(&sReceiveFrame);
+    assert(iid != INVALID_VALUE);
```

Comment:
Same as above.

Reply:
Same response as above. Please let me know what you think.